src/Backend/Core/Layout/Sass/components/_pages.scss

```diff
@@ -221,6 +221,7 @@
 
         .fa {
           padding-right: 0;
+          margin-right: 0;
```

Comment:
i don't explicitly but the full selector is   
```
.templatePositionCurrentType:hover .buttonHolder .btn .fa {}
```
`.templatePositionCurrentType` doesn't contain any buttons with text and never will